docs/core/tools/dotnet-test.md

```diff
@@ -129,7 +129,7 @@ Where `Microsoft.NET.Test.Sdk` is the test host, `xunit` is the test framework.
 
 - **`-f|--framework <FRAMEWORK>`**
 
-  Forces the use of `dotnet` or .NET Framework test host for the test binaries. This option only determines which type of host to use. The actual framework version to be used is determined by the *runtimeconfig.json* of the test project. When not specified, the [TargetFramework assembly attribute](/dotnet/api/system.runtime.versioning.targetframeworkattribute) is used to determine the type of host. When that attribute is stripped from the *.dll*, the .NET Framework host is used.
+  The [target framework moniker (TFM)](../../standard/frameworks.md) of the target framework to run tests for. The target framework must also be specified in the project file.
```

Comment:
From the [vstest docs](https://docs.microsoft.com/en-us/visualstudio/test/vstest-console-options?view=vs-2022), it looks like it might not match the TFM strings you linked to?

<html>
<body>
<!--StartFragment-->

/Framework: [framework version] | Target .NET version to be used for test execution.Example values are Framework35, Framework40, Framework45, FrameworkUap10, .NETCoreApp,Version=v1.1.TargetFrameworkAttribute is used to automatically detect this option from your assembly, and defaults to Framework40 when the attribute is not present. You must specify this option explicitly if you remove the TargetFrameworkAttribute from your .NET Core assemblies.If the target framework is specified as Framework35, the tests run in CLR 4.0 "compatibility mode".Example: /Framework:framework40
-- | --


<!--EndFragment-->
</body>
</html>/Framework: [framework version]	Target .NET version to be used for test execution.
Example values are Framework35, Framework40, Framework45, FrameworkUap10, .NETCoreApp,Version=v1.1.
[TargetFrameworkAttribute](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.versioning.targetframeworkattribute) is used to automatically detect this option from your assembly, and defaults to Framework40 when the attribute is not present. You must specify this option explicitly if you remove the TargetFrameworkAttribute from your .NET Core assemblies.
If the target framework is specified as Framework35, the tests run in CLR 4.0 "compatibility mode".
Example: /Framework:framework40

Reply:
dotnet test --framework is apparently not the same as vstest /Framework. when I run it on a multitargeted project it accepts the standard target framework monikers, the same ones used in the .csproj file.